CHANGELOG.md

```diff
@@ -4,6 +4,22 @@
     * In addition to setting `PYSPARK_SUBMIT_ARGS`, also explicitly set config params so they are picked up by an already-running JVM
     * Register a handler to stop spark session on python termination to deal with [SPARK-27927](https://issues.apache.org/jira/browse/SPARK-27927)
 * Removed `has_package` and `has_jar` functions, which are incomplete checks (resulting in false negatives) and are merely syntactic sugar.
+* Added option (class variable) `app_id_template` to autogenerate spark option `spark.app.id`,
+  with default set to: `sparkly-{process_uid}-{session_time}`.
+  This is a template-formatted string with access to four variables:
```

Comment:
We should either update process_time / uid below or maybe just remove all the "documentation" from here and only keep the first two lines `Added option ... with default set to X` in the CHANGELOG. Probably the latter is easier